__tests__/app.test.js

```diff
@@ -67,16 +69,16 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/1")
       .expect(200)
       .then(({ body }) => {
-        expect(Array.isArray(body)).toBe(false);
-        expect(typeof body).toBe("object");
+        expect(Array.isArray(body.article)).toBe(false);
+        expect(typeof body.article).toBe("object");
```

Comment:
Similar as before, I would look at toBeInstanceOf(Object) that would make like 72 redundant and not needed.